### DISCUSSION TITLE
Is it possible to route the traffic via another container?

### DISCUSSION THREAD

**Discussion by wdsgn** [RaspAP/raspap-docker#14] [General]
I have a container running a VPN client. Is it possible to create a container with raspap and route all its traffic via the VPN container? 

This way it would be very practical, because I could serve several clients with a single VPN connection. 

Thanks in advance.

**Comment by billz** [comment]
Should be possible with iptables NAT rules on the host. You could modify [these rules](https://github.com/RaspAP/raspap-docker?tab=readme-ov-file#allow-wifi-clients-to-connect-to-lan-and-internet) to route traffic through a `tun0` VPN interface, for example.
